apps/sms/views/conversation/js/information.js

```diff
@@ -339,6 +346,51 @@ var VIEWS = {
           l10nContainsDateSetup(this.receivedTimestamp, message.timestamp);
           l10nContainsDateSetup(this.sentTimestamp, message.sentTimestamp);
           setL10nAttributes(this.sentTitle, 'message-sent');
+
+          var units = {
+            months: MONTH,
+            days: DAY,
+            hours: HOUR,
+            minutes: MINUTE,
+            seconds: SECOND
+          };
+          var lateness = message.timestamp - message.sentTimestamp;
+          var showLatenessNotice = message.timestamp && message.sentTimestamp &&
+            lateness >= LATENESS_DISPLAY_LOWER_LIMIT;
+
+          if (showLatenessNotice) {
+            [
+              'months',
+              'days',
+              'hours',
+              'minutes',
+              'seconds'
+            ].forEach((unit) => {
+              var output = 0;
+              var unitElement = this[Utils.camelCase('lateness-' + unit)];
```

Comment:
I wasn't very clear in my previous comment; what do you think of doing the querySelector here (because it's used only here) instead of binding the element at init time ?